src/Umbraco.Infrastructure/Services/Implement/DataTypeService.cs

```diff
@@ -220,8 +226,7 @@ public Attempt<OperationResult> DeleteContainer(int containerId, int userId = Cm
                     _dataTypeContainerRepository.Save(container);
                     scope.Complete();
 
-                    // TODO: triggering SavedContainer with a different name?!
-                    scope.Events.Dispatch(SavedContainer, this, new SaveEventArgs<EntityContainer>(container, evtMsgs), "RenamedContainer");
+                    scope.Notifications.Publish(new EntityContainerRenamedNotification(container, evtMsgs));
```

Comment:
I get what you're doing here... I just can't help but wonder if I as a 3rd party implementer would expect "saving + saved" notifications (with an option to cancel) here? I know that's not the case as-is, but since we're redoing this whole thing... worth a consideration.

Reply:
OK will add a Moving & Moved